src/StreamToActionTranslator.h

```diff
@@ -26,27 +24,35 @@ For more information, please refer to https://unlicense.org
 #include <type_traits>
 #include <concepts>
 #include <source_location>
-
+#include <set>
+#include <unordered_map>
+#include <unordered_set>
+#include <memory>
 
 namespace sds
 {
 	namespace chron = std::chrono;
-	using Index_t = uint32_t;
+	using Index_t = uint16_t;
```

Comment:
Added for possibility Windows platform calling conventions are limited in register size and quantity compared to Linux.